web/client/test-resources/testMockComp.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
I think we could write this component directly inside the test file on top instead to create a separated one